rce/src/runtime/acquire.ts

```diff
@@ -39,5 +121,72 @@ export async function acquireRuntime() {
     }
   }
 
+  // Attempt to set the "latest" tag to true for each language
+  const languageVersions: LanguageVersion[] = [];
+  for (let i = 0; i < runtimes.length; i++) {
+    const languageIndex = languageVersions.findIndex(r => r.language === runtimes[i].language);
+    if (languageIndex === -1) {
+      languageVersions.push({ language: runtimes[i].language, versions: [{ index: i, version: runtimes[i].version }] });
+      continue;
+    }
```

Comment:
menurutku lebih jelas kalo gini

```suggestion
    const isValidRuntimeLanguage = languageVersions.findIndex(r => r.language === runtimes[i].language) === -1;
    if (isValidRuntimeLanguage) {
      languageVersions.push({ language: runtimes[i].language, versions: [{ index: i, version: runtimes[i].version }] });
      continue;
    }
```